File: 30 Days of Code/Python/Day 14 - Scope.py
class Difference:
    def __init__(self, a):
        self.__elements = a

    # Add your code here
    def computeDifference(self):
        
        e = sorted(self.__elements)
        l = len(e)-1
        self.maximumDifference = abs(e[0]-e[l])
        

# End of Difference class

_ = input()
a = [int(e) for e in input().split(' ')]

d = Difference(a)
d.computeDifference()

print(d.maximumDifference)

File: 30 Days of Code/PHP/Day 14 - Scope.php
<?php
    class Difference{
    private $elements=array();
    public $maximumDifference;

// Write your code here
    function __construct($elements) {
       $this->elements = $elements;
        
    }
    function computeDifference(){
    sort($this->elements);
    $l = count($this->elements)-1;
    $this->maximumDifference = abs($this->elements[0]-$this->elements[$l]);
    }

} //End of Difference class  
     

$N=intval(fgets(STDIN));
$a =array_map('intval', explode(' ', fgets(STDIN)));
$d=new Difference($a);
$d->ComputeDifference();
print ($d->maximumDifference);
?>